evalml/tests/data_checks_tests/test_natural_language_nan_data_check.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+import pandas as pd
+import woodwork as ww
+
+from evalml.data_checks import (
+    DataCheckError,
+    DataCheckMessageCode,
+    NaturalLanguageNaNDataCheck
+)
+
+
+def test_nl_nan_data_check_error():
+    data = pd.DataFrame({'natural_language': [None, "string_that_is_long_enough_for_natural_language", "string_that_is_long_enough_for_natural_language"]})
+    nl_nan_check = NaturalLanguageNaNDataCheck()
+    assert nl_nan_check.validate(data) == {
+        "warnings": [],
+        "actions": [],
+        "errors": [DataCheckError(message='Input natural language column(s) (natural_language) contains NaN values. Please impute NaN values or drop these rows or columns.',
+                                  data_check_name=NaturalLanguageNaNDataCheck.name,
+                                  message_code=DataCheckMessageCode.NATURAL_LANGUAGE_HAS_NAN,
+                                  details={"columns": 'natural_language'}).to_dict()]
+    }
+
+
+def test_nl_nan_data_check_error_no_nan():
+    nl_nan_check = NaturalLanguageNaNDataCheck()
+    assert nl_nan_check.validate(pd.DataFrame({'natural_language': ["string_that_is_long_enough_for_natural_language", "string_that_is_long_enough_for_natural_language"]})) == {
+        "warnings": [],
+        "actions": [],
+        "errors": []
+    }
+
+
+def test_nl_nan_data_check_error_other_cols_with_nan():
+    data = pd.DataFrame(np.random.randint(0, 10, size=(10, 4)))
```

Comment:
Is the purpose of this test to show that it doesn't throw any errors if there are `NaN` values in a dataset without `Natural Language` columns? It seems like it'd be more useful if you included a NL column with this (without nulls) to make sure there are no errors raised when other columns have `NaN`s.